systemtest/src/test/java/io/strimzi/systemtest/cruisecontrol/CruiseControlST.java

```diff
@@ -79,9 +81,28 @@ void testAutoCreationOfCruiseControlTopics(ExtensionContext extensionContext) {
                 .editKafka()
                     .addToConfig("auto.create.topics.enable", "false")
                 .endKafka()
+                .editCruiseControl()
+                    .withResources(new ResourceRequirementsBuilder()
+                        .addToLimits("memory", new Quantity("256Mi"))
+                        .addToLimits("cpu", new Quantity("1"))
+                        .addToRequests("memory", new Quantity("256Mi"))
+                        .addToRequests("cpu", new Quantity("0.8"))
+                        .build())
+                    .withNewJvmOptions()
+                        .withXmx("512M")
```

Comment:
you cannot fit Xmx512M into 256Mi. You probably cannot fit -Xms256M either.

Reply:
Aaah, I see now. Of course it's connected to container limits.